dialects/mysql/src/testFixtures/resources/fixtures_mysql/alter-table-rename-index/1.s

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE animals (
+    id BIGINT AUTO_INCREMENT,
+    name VARCHAR(30) NOT NULL,
+    species VARCHAR(30) NOT NULL,
+    UNIQUE KEY unq_name (name),
+    KEY idx_species (species)
+);
+
+ALTER TABLE animals
+  RENAME INDEX `unq_name` TO `unq_animals_name`,
```

Comment:
can you also add a second statement to this test which uses the new name of the index, like a drop index statement. Just want to make sure that you can reference the new name fine

Reply:
Good point. Added